sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -474,6 +479,55 @@ case class Lag(input: Expression, offset: Expression, default: Expression)
   override val direction = Descending
 }
 
+/**
+ * The NthValue function returns the value of `input` at the row that is the `offset`th row of
+ * the window frame (counting from 1). Offsets start at 0, which is the current row. When the
```

Comment:
`Offsets start at 0, which is the current row.` do you mean first row?

Reply:
and it seems contradict with `(counting from 1)`

Reply:
Sorry! I made a mistake.
`Offsets start at 1`